testing/wine-mono/APKBUILD

```diff
@@ -0,0 +1,32 @@
+# Maintainer: Jens Staal <staal1978@gmail.com>
+pkgname=wine-mono
+pkgver=4.6.3
+pkgrel=0
+pkgdesc="Wine's built-in replacement for Microsoft's .NET Framew"
+url="http://wiki.winehq.org/Mono"
+arch="any"
```

Comment:
This is not a valid type. If this binary is platform-independent, then it should be `noarch`.